pkg/server/api_test.go

```diff
@@ -18,49 +19,90 @@ func (ms *mockServer) GetConfig(pr poolRequest) (*ignv2_2types.Config, error) {
 }
 
 type scenario struct {
-	name           string
+	request        *http.Request
 	expectedStatus int
 	serverFunc     func(poolRequest) (*ignv2_2types.Config, error)
 }
 
 func TestAPIHandler(t *testing.T) {
 	scenarios := []scenario{
 		{
-			name:           "not-found",
```

Comment:
can we keep the `name` these are useful to users looking at the result for a test.
`fmt.Sprintf("%s %s", scenario.request.Method, scenario.request.URL.Path)` leaves more to be desired...

Reply:
> can we keep the `name`...

Actually, I think the method/path thing is more readable ;).  But I've taken a stab at restoring names with 22d0c45 -> 8c9980d.